packages/trimerge-sync-indexed-db/src/IndexedDbCommitRepository.ts

```diff
@@ -246,16 +236,26 @@ export class IndexedDbCommitRepository<CommitMetadata, Delta, Presence>
     }
     const db = await createIndexedDb<CommitMetadata, Delta>(this.dbName);
     db.onclose = () => {
-      this.db = this.connect(true);
+      this.dbInfo = this.connect(true);
     };
-    return db;
+    const readTx = db.transaction(['remotes'], 'readonly');
+    const remotes = readTx.objectStore('remotes');
+    const remote = (await remotes.get(this.remoteId)) ?? {};
+    if (!remote.localStoreId) {
+      await readTx.done;
+      remote.localStoreId = await this.localIdGenerator();
```

Comment:
Based on my reading of the [`idb` docs](https://www.npmjs.com/package/idb), we're safe awaiting this because we have no open transactions, right?